doc/sphinx/changes.rst

```diff
@@ -8,6 +8,687 @@ Recent changes
+- **Changed:**
+  PrimFloat notations now match up with the rest of the standard library: :g:`m
+  == n`, :g:`m < n`, and :g:`m <= n` have been replaced with :g:`m =? n`, :g:`m
+  <? n`, and :g:`m <=? n`.  The old notations are still available as deprecated
+  notations.  Additionally, there is now a
+  ``Coq.Floats.PrimFloat.PrimFloatNotations`` module that users can import to
+  get the ``PrimFloat`` notations without unqualifying the various primitives
+  (`#12556 <https://github.com/coq/coq/pull/12556>`_, fixes `#12454
+  <https://github.com/coq/coq/issues/12454>`_, by Jason Gross).
+- **Changed:**
+  Change the sort of cyclic numbers from Type to Set. For backward compatibility, a dynamic sort was defined in the 3 packages bignums, coqprime and color.
+  See for example commit 6f62bda in bignums.
+  (`#12801 <https://github.com/coq/coq/pull/12801>`_,
+  by Vincent Semeria).
+- **Changed:**
+  ``Require Import Coq.nsatz.NsatzTactic`` now allows using :tacn:`nsatz`
+  with `Z` and `Q` without having to supply instances or using ``Require Import Coq.nsatz.Nsatz``, which
+  transitively requires unneeded files declaring axioms used in the reals
+  (`#12861 <https://github.com/coq/coq/pull/12861>`_,
+  fixes `#12860 <https://github.com/coq/coq/issues/12860>`_,
+  by Jason Gross).
+- **Deprecated:**
+  ``prod_curry`` and ``prod_uncurry``, in favor of ``uncurry`` and ``curry``
+  (`#12716 <https://github.com/coq/coq/pull/12716>`_,
+  by Yishuai Li).
+- **Added:**
+  New lemmas about ``repeat`` in ``List`` and ``Permutation``: ``repeat_app``, ``repeat_eq_app``, ``repeat_eq_cons``, ``repeat_eq_elt``, ``Forall_eq_repeat``, ``Permutation_repeat``
+  (`#12799 <https://github.com/coq/coq/pull/12799>`_,
+  by Olivier Laurent).
+- **Added:**
+  Extend some list lemmas to both directions: `app_inj_tail_iff`, `app_inv_head_iff`, `app_inv_tail_iff`.
+  (`#12094 <https://github.com/coq/coq/pull/12094>`_,
+  fixes `#12093 <https://github.com/coq/coq/issues/12093>`_,
+  by Edward Wang).
+- **Added:**
+  ``Decidable`` instance for negation
+  (`#12420 <https://github.com/coq/coq/pull/12420>`_,
+  by Yishuai Li).
+- **Fixed:**
+  `Coq.Program.Wf.Fix_F_inv` and `Coq.Program.Wf.Fix_eq` are now axiom-free. They no longer assume proof irrelevance.
+  (`#13365 <https://github.com/coq/coq/pull/13365>`_,
+  by Li-yao Xia).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coq's core system now uses the `zarith <https://github.com/ocaml/Zarith>`_
+  library, based on GNU's gmp instead of ``num`` which is
+  deprecated upstream. The custom ``bigint`` module is
+  not longer provided; note that ``micromega`` still uses
+  ``num``
+  (`#11742 <https://github.com/coq/coq/pull/11742>`_,
+  by Emilio Jesus Gallego Arias and Vicent Laporte).
+- **Removed:**
+  The `num` library is not linked to Coq anymore
+  (`#13007 <https://github.com/coq/coq/pull/13007>`_,
+  by Emilio Jesus Gallego Arias).
```

Comment:
The two entries should be merged into a single one, saying that `num` is not a dependency of Coq anymore (not just the core system) and it doesn't need to even mention micromega. BTW, shouldn't Frédéric Besson be credited as well? (e.g., "with help from").

Reply:
Fixed.